cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,82 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
+import accountPageObject from '../support/pages/account.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+
+const settingsPage = newSettingsPageObject();
+const homePage = newhomePageObject();
+const accountPage = newaccountPageObject();
```

Comment:
There is a typo in the instantiation of `accountPageObject`. It should be `new accountPageObject()` instead of `newaccountPageObject()`.